### DISCUSSION TITLE
Is it possible to use more than one server?

### DISCUSSION THREAD

**Discussion by Nauseouswave** [Taxel/PlexTraktSync#693] [Q&A] [answered]
I use more than one server and would like to know if this tool can track all of them.

Or would running multiple instances of it to achieve my goal is the only solution?

**Comment by mvanbaak** [comment, marked as answer]
Running an instance for each server is the only way to handle this.
With current versions you dont have to install PlexTraktSync multiple times anymore. Just make different configuration directories, one for every user+server combination, and use those.

See https://github.com/Taxel/PlexTraktSync/issues/629 and https://github.com/Taxel/PlexTraktSync/issues/580 for more info

**Reply by glensc** [reply]
You can also use `--server` command line option to choose a different server. config and logs can be shared.
